webapp/commonComponents/validationTooltip.js

```diff
@@ -11,7 +11,13 @@ export default ({ validation, className, showKeys, children }) => {
   const i18n = useI18n()
 
   const isValid = Validation.isValid(validation)
-  const type = isValid ? '' : 'error'
+
+  const type = isValid
+    ? ''
+    : Validation.hasWarnings(validation) || Validation.hasWarningsInFields(validation)
```

Comment:
```Validation.hasWarnings(validation) || Validation.hasWarningsInFields(validation) ``` is used several times. can we have a method  ```Validation.isWarning```?

Reply:
ok